openadapt/app/dashboard/.gitignore

```diff
@@ -0,0 +1,35 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Is there a reason to have multiple `.gitignore`s, or should the contents of this file go in the root `.gitignore`?

Reply:
Felt like with this being a pretty sizeable project on its own, it made sense for it to have its own `.gitignore`